src/frontend/utils/testing.tsx

```diff
@@ -195,28 +214,36 @@ const wrapMedDefaultProvidere = (children: ReactNode, språkTekster: Record<stri
             AppProvider,
             EøsProvider,
             RoutesProvider,
+            MemoryRouter,
             StegProvider,
             AppNavigationProvider,
         ],
+        mocketNettleserHistorikk,
         children,
         språkTekster
     );
 
-export const TestProvidere: React.FC<{ tekster?: Record<string, string> }> = ({
-    tekster,
-    children,
-}) => wrapMedDefaultProvidere(children, tekster ?? {});
+export const TestProvidere: React.FC<{
+    tekster?: Record<string, string>;
+    mocketNettleserHistorikk?: string[];
+    children?: ReactNode;
+}> = ({ tekster, mocketNettleserHistorikk = ['/'], children }) =>
+    wrapMedDefaultProvidere(children, tekster ?? {}, mocketNettleserHistorikk);
 
-export const TestProvidereMedEkteTekster: React.FC = ({ children }) => (
-    <TestProvidere tekster={norskeTekster}>{children}</TestProvidere>
+export const TestProvidereMedEkteTekster: React.FC<{
+    mocketNettleserHistorikk?: string[];
+    children?: ReactNode;
+}> = ({ mocketNettleserHistorikk, children }) => (
+    <TestProvidere
+        tekster={norskeTekster}
+        children={children}
+        mocketNettleserHistorikk={mocketNettleserHistorikk}
+    />
 );
 
-export const mockHistory = (
-    newHistory: string[]
-): { mockedHistory: History; mockedHistoryArray: string[] } => {
-    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-    // @ts-ignore denne har vi definert i __mocks__/history
-    return history.__setHistory(newHistory);
+export const LesUtLocation = () => {
```

Comment:
Hjelpekomponent for å lese ut location - inspirert av hvordan dette løses i [testene til react-router](https://github.com/remix-run/react-router/blob/main/packages/react-router/__tests__/useNavigate-test.tsx)